### ISSUE TITLE
How to Customize Built-in Research Recipes?

### ISSUE THREAD

**Issue by xiakkto** [mcenderdragon/Futurepack-API#995]
Although there is a feature in the config to disable research, I want to modify my modpack while keeping this feature. However, the built-in research recipes seem to be non-customizable. So, is it possible to make these existing research recipes editable by us? For example, by using a Datapack?

**Comment by Ashwinaaaaaaahdt** [comment]
Those modifications of recipes are easy to make ,the only thing you need to do is to open futurepack .jar files by 7zip and go to
/Data/recipes
There are files called JSON files where you can open it and you can edit the recipes
And replace the items
That is just simple way to do the work

**Comment by Ashwinaaaaaaahdt** [comment]
> Although there is a feature in the config to disable research, I want to modify my modpack while keeping this feature. However, the built-in research recipes seem to be non-customizable. So, is it possible to make these existing research recipes editable by us? For example, by using a Datapack?

Those modifications of recipes are easy to make ,the only thing you need to do is to open futurepack .jar files by 7zip and go to
/Data/recipes if you want to edit recipes 
There are files called JSON files where you can open it and you can edit the recipes
And replace the items
That is just simple way to do the work

**Comment by xiakkto** [comment]
> > Although there is a feature in the config to disable research, I want to modify my modpack while keeping this feature. However, the built-in research recipes seem to be non-customizable. So, is it possible to make these existing research recipes editable by us? For example, by using a Datapack?
> 
> Those modifications of recipes are easy to make ,the only thing you need to do is to open futurepack .jar files by 7zip and go to /Data/recipes if you want to edit recipes There are files called JSON files where you can open it and you can edit the recipes And replace the items That is just simple way to do the work

No, what I mean is modifying the research items in "research", for example, the fluid pipe requires a milk bucket, but I can't figure out how to modify the research method for this item.

**Comment by Ashwinaaaaaaahdt** [comment]
Go to /futurepack/futurepack/common/research and edit the reserch recipes

**Comment by xiakkto** [comment]
> Go to /futurepack/futurepack/common/research and edit the reserch recipes

But I can't customize this recipe within the datapack.

**Comment by Ashwinaaaaaaahdt** [comment]
> > Go to /futurepack/futurepack/common/research and edit the reserch recipes
> 
> But I can't customize this recipe within the datapack.

Make the edit by using 7zip and use notepad
Firstly make "only readable" to false by going to properties of file by right clicking it in file explorer 
And I wasn't mostly active and that's the reason for being late in answering
